doc/examples/segment_quickbundles.py

```diff
@@ -33,17 +33,68 @@
 streamlines = [i[0] for i in streams]
 
 """
-Perform QuickBundles clustering with a 10mm distance threshold after having
-downsampled the streamlines to have only 12 points.
+Perform QuickBundles clustering using the MDF metric and a 10mm distance
+threshold. Since the MDF metric requires streamlines to have the same number
+of points, we first downsample them so they have only 18 points and then run
+the clustering algorithm.
 """
 
-qb = QuickBundles(streamlines, dist_thr=10., pts=18)
+qb = QuickBundles(threshold=10.)
+streamlines = set_number_of_points(streamlines, nb_points=18)
+clusters = qb.cluster(streamlines)
 
 """
-qb has attributes like `centroids` (cluster representatives), `total_clusters`
-(total number of clusters) and methods like `partitions` (complete description
-of all clusters) and `label2tracksids` (provides the indices of the streamlines
-which belong in a specific cluster).
+`clusters` is a `ClusterMap` object which contains intuitive fonctionnalies to
+interact with the clustering result.
+"""
+
+print("Nb. clusters:", len(clusters))
+print("Cluster sizes:", map(len, clusters))
+print("Small clusters:", clusters < 10)
+print("Streamlines indices of the first cluster:\n", clusters[0].indices)
+print("Centroid of the last cluster:\n", clusters[-1].centroid)
+
```

Comment:
I added `get_size()`, `get_clusters_sizes()`, `get_large_clusters(min_size)` and `get_small_clusters(max_size)`.